Tests/SonarScanner.MSBuild.Common.UnitTests/EnvScannerPropertiesProviderTest.cs

```diff
@@ -38,20 +37,22 @@ public void ParseValidJson()
             provider.GetAllProperties().First().Value.Should().Be("http://myhost");
         }
 
-        [TestCleanup]
-        public void Cleanup()
-        {
-            Environment.SetEnvironmentVariable("SONARQUBE_SCANNER_PARAMS", null);
```

Comment:
The test shouldn't set the variable to null - it should restore whatever value existed before the test started (which is what the _EnvironmentVariableScope_ does when it is disposed).